### PR TITLE
make clean: remove translations/*.h

### PR DESCRIPTION
**Pull request by griebd**
Just a small update to remove the generated .h translation files on make clean.
If there is a reason to not delete then on make clean just ignore this.

~~Added bitmaps/Taranis/mainmenu.png to the clean list too. It's auto-generated.~~

### CODE REVIEW

File: radio/src/Makefile [hunk 1]
```diff
@@ -1801,9 +1801,11 @@ clean_list :
 	$(REMOVE) bitmaps/*.lbm
 	$(REMOVE) bitmaps/*/*.lbm
 	$(REMOVE) bitmaps/*/*/*.lbm
+	$(REMOVE) bitmaps/Taranis/mainmenu.png
```
Comment (bsongis) [inline radio/src/Makefile]: This one should come in another commit, which removes the code I wrote for it, as it won't be useful anymore

**Comment by griebd** [comment]
@bsongis, as I don't know with code you refer to I will leave the other change to you. Is that what you mean, right?

**Comment by bsongis** [comment]
Yes I prefer you don't remove this line now

**Comment by projectkk2glider** [comment]
Sorry I made your proposed changes in my commits, but it was easier just to use one line from your commit than to resolve merge conflicts. Thanks.

**Comment by griebd** [comment]
no problems at all! :wink:

**Pull request closed without merging**